### DISCUSSION TITLE
CRUD list items

### DISCUSSION THREAD

**Discussion by TihonchukArtyom89** [OrchardCMS/OrchardCore#9916] [General]
Hello everyone! I want to make a simple application on Orchard Core CMS with list of custom content items(advertisments) and different role permissions(moderators, user) for content activity(moderator can edit all list item, created by all user, user can edit list item? created by only his account).  At now I make data structure of list of advertisments and two page on project: 
-Page which show all advertisments
-Page which show selected advertisment
I`m stuck on the stage, where need make the CRUD operations on advertisments. Could anyone explain me, how to do this in Orchard Core? 
Kind regards.

**Comment by Skrypt** [comment]
You're supposed to be able to do all of this by simply using the Content Items list in OC admin. You're supposed to only need to use the ContentManager class to do this.

**Reply by TihonchukArtyom89** [reply]
Would you like to show an example of using this class ?

**Reply by Skrypt** [reply]
You can look in the OrchardCore.Contents module in the AdminController.cs

**Reply by ns8482e** [reply]
There's sample in Demo project

**Reply by TihonchukArtyom89** [reply]
Thank you for the helpful advice)
